app/controllers/hub/documents_controller.rb

```diff
@@ -17,12 +17,15 @@ def index
     end
 
     def show
+      log_document_access!
       redirect_to transient_storage_url(@document.upload.blob)
     end
 
     def new; end
 
-    def edit; end
+    def edit
+      log_document_access!
+    end
```

Comment:
I added this in response to https://github.com/codeforamerica/vita-min/pull/901#issuecomment-792976981

Reply:
This seems unnecessary; is peaceful I suppose to keep! The document URL doesn't actually get sent to any web browsers through this method.